### PR TITLE
Feature/revised csv format

### PR DESCRIPTION
**Pull request by AndrewKeig**

### CODE REVIEW

File: server/models/BulkImport/csv/workers.js [hunk 1]
```diff
@@ -1686,6 +1535,59 @@ class Worker {
     this._qualifications = myProcessedQualifications.filter(thisQualification => thisQualification !== null && thisQualification !== false);
   }
 
+  _validateAmhp() {
+    const amhpValues = [1,2,999];
+    const myAmhp = parseInt(this._currentLine.AMHP);
+    const socialWorkerJobRole = 6;
+
+    if (this._currentLine.AMHP && this._currentLine.AMHP.length > 0) {
+      if (this._mainJobRole === socialWorkerJobRole ) {
```
Comment (wozzer72) [inline server/models/BulkImport/csv/workers.js]: We don't do cross validations yet. This needs to be handled in the API as it's not specific to bulk upload

File: server/models/BulkImport/csv/workers.js [hunk 2]
```diff
@@ -1686,6 +1535,59 @@ class Worker {
...
+      else {
+        switch (myAmhp) {
+          case 1:
+            this._amhp = 'Yes';
+            break;
+          case 2:
+            this._amhp = 'No';
+            break;
+          case 999:
+            this._amhp = 'Not known';
```
Comment (wozzer72) [inline server/models/BulkImport/csv/workers.js]: This needs to be "Don't know" - check `server/models/classes/worker/properties/approvedMentalHealthWorkerProperty.js` for expected values.

File: server/models/BulkImport/csv/workers.js [hunk 3]
```diff
@@ -119,6 +115,7 @@ class Worker {
   static get QUAL_ACH02_NOTES_ERROR() { return 5070; }
   static get QUAL_ACH03_ERROR() { return 5080; }
   static get QUAL_ACH03_NOTES_ERROR() { return 5090; }
+  static get AMHP_ERROR() { return 6000; }
```
Comment (wozzer72) [inline server/models/BulkImport/csv/workers.js]: Not specific to this field, but gender type mapping of "4" to "Other" is missing

File: server/models/BulkImport/csv/workers.js [hunk 4]
```diff
@@ -462,7 +450,7 @@ class Worker {
   }
 
   _validateGender() {
-    const genderValues = [1,2,3]; //[MALE=1, FEMALE=2, UNKNOWN=3];
+    const genderValues = [1,2,3,4]; //[MALE=1, FEMALE=2, UNKNOWN=3, OTHER=4];
```
Comment (wozzer72) [inline server/models/BulkImport/csv/workers.js]: Missing the mapping of "4" to other; you couldn't have confirmed this was presented in the entity.

File: server/models/BulkImport/csv/workers.js [hunk 5]
```diff
@@ -620,7 +608,7 @@ class Worker {
   }
 
   _validateDisabled() {
-    const disabledValues = [0,1]; 
+    const disabledValues = [0,1,2,3]; 
     const myDisabled = parseInt(this._currentLine.DISABLED, 10);
 
     // optional
```
Comment (wozzer72) [inline server/models/BulkImport/csv/workers.js]: Missing the mapping of 2 and 3 to `Don't know` and `Other`

File: server/models/BulkImport/csv/workers.js (hunk 1 above)
Comment (AndrewKeig) [reply server/models/BulkImport/csv/workers.js]: Should I move this, or just remove it

Comment (AndrewKeig) [reply server/models/BulkImport/csv/workers.js]: Removed.

File: server/models/BulkImport/csv/workers.js (hunk 2 above)
Comment (AndrewKeig) [reply server/models/BulkImport/csv/workers.js]: Fixed.

File: server/models/BulkImport/csv/workers.js (hunk 3 above)
Comment (AndrewKeig) [reply server/models/BulkImport/csv/workers.js]: Added

File: server/models/BulkImport/csv/workers.js (hunk 4 above)
Comment (AndrewKeig) [reply server/models/BulkImport/csv/workers.js]: Added

File: server/models/BulkImport/csv/workers.js (hunk 5 above)
Comment (AndrewKeig) [reply server/models/BulkImport/csv/workers.js]: Added

**Pull request merged**